tests/beaker_tests/cisco_bgp_neighbor_af/test_bgpneighboraf.rb

```diff
@@ -529,7 +592,9 @@ def build_manifest_bgp_nbr_af(tests, id)
                tests[id][:title_pattern])
   tests[id][:manifest] = "cat <<EOF >#{PUPPETMASTER_MANIFESTPATH}
   node 'default' {
-    cisco_bgp_neighbor_af { '#{tests[id][:title_pattern]}':
+    #{extra_config}
+
+    cisco_bgp_neighbor_af { '#{af[:asn]} #{af[:vrf]} #{af[:neighbor]} #{af[:afi]} #{af[:safi]}':
```

Comment:
I don't understand. The whole point of section 3 is testing the specified title_pattern value - by ignoring it you are defeating the point of the test, are you not?